internal/internal_task_handlers.go

```diff
@@ -115,8 +115,10 @@ type (
 		isWorkflowCompleted bool
 		result              *commonpb.Payloads
 		err                 error
-
+		// previousStartedEventID is the event ID of the workflow task started of the previous workflow task.
 		previousStartedEventID int64
+		// lastHandledEventID is the event ID of the last command that the workflow state machine processed.
+		lastHandledEventID int64
```

Comment:
```suggestion
		// lastHandledEventID is the event ID of the last command-event that the workflow state machine processed.
		lastHandledEventID int64
```

Reply:
Is this actually true though? It looks like this is set for _all_ processed events, not just command-events

Reply:
Yeah I didn't mean command event here, but saying event would be more clear